lcls_naming_tool/fungible_element_taxon.json

```diff
@@ -0,0 +1,111 @@
+{
+    "4FCCM": "4-F Channel Cut Mono",
+	"4FFOCUS": "Mirror 4F Focus ",
+	"ATM": "Arrival time monitor",
+	"BCS": "Beam containment stopper",
+	"BTM": "Burn Thru Monitor",
+	"BEND": "Bendable mirror, using piezo benders",
+	"C1": "",
```

Comment:
What is our policy on these Fungibles that have no description?  Should they be considered invalid?  (This is not a problem with this PR, but rather the naming convention)

Reply:
The current naming tool doesn't use any json descriptions, but need to clarify policy on these fungibles in next PR.